dmk2cw.c

```diff
@@ -358,15 +358,19 @@ approx(int a, int b)
 #define MFM_GAP3Z 8 /*12 nominal, but that is too many.  3 is too few! */
 #define FM_GAP3Z  4 /*6 nominal */
 
-/* Byte encodings */
-#define SKIP    0  /* padding byte in FM area of a DMK */
-#define FM      1  /* FM with FF clock */
-#define FM_IAM  2  /* FM with D7 clock (IAM) */
-#define FM_AM   3  /* FM with C7 clock (IDAM or DAM) */
-#define MFM     4  /* MFM with normal clocking algorithm */
-#define MFM_IAM 5  /* MFM C2 with missing clock */
-#define MFM_AM  6  /* MFM A1 with missing clock */
-#define RX02    7  /* DEC-modified MFM as in RX02 */
+/* Byte encodings and letters used when logging them */
+#define SKIP    0  /* -  padding byte in FM area of a DMK */
+#define FM      1  /* F  FM with FF clock */
+#define FM_IAM  2  /* I  FM with D7 clock (IAM) */
+#define FM_AM   3  /* A  FM with C7 clock (IDAM or DAM) */
+#define MFM     4  /* M  MFM with normal clocking algorithm */
+#define MFM_IAM 5  /* J  MFM C2 with missing clock */
+#define MFM_AM  6  /* B  MFM A1 with missing clock */
+#define RX02    7  /* X  DEC-modified MFM as in RX02 */
+#define encoding_letter "-FIAMJBX"
+
+/* Or'ed into encoding at end of sector data */
+#define SECTOR_END 8
```

Comment:
Not sure why SECTOR_END is in base 10 and made to appear in sequence with the other defines SKIP thorugh RX02.  SECTOR_END is a bitfield, so I'd prefer to see it in base 16.  Also, why not make SECTOR_END an MSB with a value of 0x80, that way if other encodings are added SECTOR_END doesn't have to be changed?